src/components/BlacklistedTagsSelector/index.tsx

```diff
@@ -124,17 +124,23 @@ const ControlledKeywordSelector = ({
 
       const keywords = await Promise.all(
         defaultValue.split(',').map(async (keywordId) => {
-          const { data } = await axios.get<Keyword>(
-            `/api/v1/keyword/${keywordId}`
-          );
-          return data;
+          try {
+            const { data } = await axios.get<Keyword>(
+              `/api/v1/keyword/${keywordId}`
+            );
+            return data;
+          } catch (error) {
+            return null;
+          }
```

Comment:
With null returned if the tag doesn't exist anymore, this wouldn't be necessary anymore